source/common/protobuf/utility.h

```diff
@@ -310,6 +310,14 @@ class DurationUtil {
    * @throw OutOfRangeException when duration is out-of-range.
    */
   static uint64_t durationToSeconds(const ProtobufWkt::Duration& duration);
+
+  /**
+   * Writes a time_point<system_clock> (SystemTime) to a protobuf Timestamp, by way of time_t.
```

Comment:
I probably wouldn't use the "write" verb here; maybe `systemClockToTimestamp` or `systemClockToTimestampConvert` would be more consistent with other methods in this file.